lighthouse-core/config/config.js

```diff
@@ -22,6 +22,24 @@ const {requireAudits, mergeOptionsOfItems, resolveModule} = require('./config-he
 /** @typedef {typeof import('../gather/gatherers/gatherer.js')} GathererConstructor */
 /** @typedef {InstanceType<GathererConstructor>} Gatherer */
 
+/** @type {Array<keyof LH.BaseArtifacts>} */
+const BASE_ARTIFACT_NAMES = [
+  'fetchTime',
+  'LighthouseRunWarnings',
+  'TestedAsMobileDevice',
+  'HostUserAgent',
+  'NetworkUserAgent',
+  'BenchmarkIndex',
+  'WebAppManifest',
+  'Stacks',
+  'traces',
+  'devtoolsLogs',
+  'settings',
+  'URL',
+  'Timing',
+  'PageLoadError',
+];
```

Comment:
I couldn't think of a better way to do this. Suggestions? We don't have a place where we export it anywhere. We could separate construction from populating in gather-runner, I guess...

Reply:
```ks
Object.keys(/** @type {Record<keyof LH.BaseArtifacts, string>} */ ({
...
}))
```

to enable type checking at least so it's automated to update?

Reply:
> to enable type checking at least so it's automated to update?

haha, it's terrible but I love it :)